src/js/express-xr/src/components/SGCharacter.js

```diff
@@ -494,7 +494,7 @@ const SGCharacter = React.memo(({ id, type, worldScale, isSelected, updateObject
           >
             <primitive userData={{ id, type }} object={skinnedMesh} />
             <primitive object={armatures[0]} />
-            {props.children}
+            {bonesHelper && <group>{props.children}</group>}
```

Comment:
Do the `children` need to be in a `Group`?

Reply:
Tried without  `Group` and doesn't seem to change anything!

Reply:
```suggestion
            {bonesHelper && props.children}
```